examples/index.html

```diff
@@ -153,7 +153,7 @@ <h1><a href="https://threejs.org">three.js</a></h1>
 
 			};
 
-			exitSearchButton.onclick = function ( ) {
+			clearSearchButton.onclick = function ( ) {
```

Comment:
It seems the input field in the examples and the manual still looses the focus.

Reply:
good catch, should be ok now